README.md

```diff
@@ -68,11 +31,55 @@ terraform plan
 terraform apply
 ```
 
-## Off boarding
+The configuration file should contain the following module definitions:
+
+### Once per account and region
+
+The `region-base` module needs to be created _once_ per account and region.
+
+```hcl
+module "<REGION_NAME>-base" {
+  source = "git::https://github.com/doitintl/terraform-eks-lens.git//region-base"
+}
+```
+
+The region and account ID are [inferred](https://github.com/doitintl/terraform-eks-lens/blob/main/region-base/main.tf#L5-L6) from your AWS provider configuration.
+Make sure that your AWS provider is configured to use the correct account and region.
+
+### Per cluster
+
+The `cluster` module needs to be created once _per cluster_.
+
+```hcl
+module "<REGION>-<CLUSTER_NAME>" {
+  source = "git::https://github.com/doitintl/terraform-eks-lens.git//cluster"
+
+  cluster = {
+    cluster_name     = "<CLUSTER_NAME>"
+    deployment_id    = "<DEPLOYMENT_ID>"
+    kube_state_image = "registry.k8s.io/kube-state-metrics/kube-state-metrics:v2.9.2" # make sure to use the latest available image
+    otel_image       = "otel/opentelemetry-collector-contrib:0.83.0" # make sure to use the latest available image
+  }
+  # If running in EKS:
+  cluster_oidc_issuer_url = "<CLUSTER_OIDC_ISSUER_URL>"
+  # Alternatively, if managing your own cluster on EC2, set `cluster_oidc_issuer_url` to an empty string and uncomment the following:
+  #ec2_cluster = true
+
+  # when configuring multiple providers for different clusters, you can configure the module to use to correct provider alias:
+  providers = {
+    kubernetes = kubernetes.<PROVIDER_ALIAS>
+  }
```

Comment:
Isn't this _required_ as soon as they have more than one cluster per region? Since there will need to be one provider per cluster.

Reply:
Technically, but you could have a single cluster module and a single kubernetes provider which will be used by default.

This configuration is generated by us anyway, so I suppose it would contain the provider name already? The readme is more of a reference.

I'm open to suggestions if you have a better idea on how to convey this.